modules/logging-forward-splunk.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+// cluster-logging-external.adoc
+//
+
+:_content-type: PROCEDURE
+[id="logging-forward-splunk_{context}"]
+= Forwarding logs to Splunk
+
+You can forward logs to the link:https://docs.splunk.com/Documentation/Splunk/9.0.0/Data/UsetheHTTPEventCollector[Splunk HTTP Event Collector (HEC)] in addition to, or instead of, the internal default {product-title} log store.
+
+[NOTE]
+====
+Using this feature with Fluentd is not supported.
+====
+
+.Prerequisites
+* Red Hat OpenShift Logging Operator 5.6 and higher
+* ClusterLogging instance with vector specified as collector
+* Base64 encoded Splunk HEC token
+
+.Procedure
+
+. Create a secret using your Base64 encoded Splunk HEC token.
+[source,terminal]
+----
+$ oc -n openshift-logging create secret generic vector-splunk-secret --from-literal hecToken=<HEC_Token>
+----
+
+. Create or edit the `ClusterLogForwarder` Custom Resource (CR) using the template below:
++
+[source,yaml]
+----
+  apiVersion: "logging.openshift.io/v1"
+  kind: "ClusterLogForwarder"
+  metadata:
+    name: "instance"
+    namespace: "openshift-logging"
+  spec:
+    outputs:
+      - name: splunk-receiver
+        secret:
+          name: vector-splunk-secret
+        type: splunk
+        url: <http://your.splunk.hec.url:8088>
+    pipelines:
+      - inputRefs:
+          - application
+          - infrastructure
+        name: <1>
+        outputRefs:
+          - splunk-receiver
+----
+
+<1> This value is optional.
```

Comment:
Do you want to go ahead and add these remaining items to the yaml, similar to what we did for Cloudwatch? 
  
<1> The name of the `ClusterLogForwarder` CR must be `instance`.
<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
<3> Specify a name for the output.
<4> Specify the name of the secret that contains your HEC token.
<5> Specify the `splunk` type.
<6> Specify the url (including port) of your splunk HEC.
<7> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
<8> Optional: Specify a name for the pipeline.
<9> Specify the name of the output to use when forwarding logs with this pipeline.

Reply:
Similar to this:   https://github.com/openshift/openshift-docs/blob/main/modules/cluster-logging-collector-log-forward-cloudwatch.adoc

Reply:
Good call, I went off the GCP one which has less details for some reason.